test/parallel/test-worker-fshandles-closed.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs/promises');
+const {
+  setImmediate,
+} = require('timers/promises');
+const { isMainThread, parentPort, Worker } = require('worker_threads');
+
+const MAX_ITERATIONS = 20;
+const MAX_THREADS = 10;
+
+if (isMainThread) {
+  function spinWorker(iter) {
+    const w = new Worker(__filename, { workerData: iter });
```

Comment:
```suggestion
    const w = new Worker(__filename);
```
The worker doesn't use this data